javascript/npm/draco3d/draco_nodejs_example.js

```diff
@@ -16,7 +16,7 @@
 
 const fs = require('fs');
 const assert = require('assert');
-const draco3d = require('./draco3d');
+const draco3d = fs.existsSync('./draco3d') ? require('./draco3d') : require('draco3d');
```

Comment:
Thanks for the fix. You are right. But I think we don't need to check existsSync. Just the following should be fine for both cases:
~~~
const draco3d = require('draco3d');
~~~